src/sonic-py-common/sonic_py_common/device_info.py

```diff
@@ -687,16 +693,33 @@ def run_command_pipe(cmd0, cmd1, cmd2):
         err = out
     return (out, err)
 
+def _modify_mac_for_asic(mac, namespace=None):
+    if namespace is None:
+        return mac
+    if namespace in ["asic0", "asic1", "asic2"]:
```

Comment:
can we get the namespace list iso hardcoding the asic names

Reply:
Fixed..